coalib/output/ConsoleInteraction.py

```diff
@@ -218,19 +219,47 @@ def print_result(console_printer,
                          "not a valid derivative of the coala result "
                          "class.")
         return
+    if section is None:
+        raise ValueError("Section has to be intializied.")
```

Comment:
if you want this, this has to be added in an own commit and please use enforce_signature for this one, it shouldn't ever happen and thus is a programmer - not runtime - error